.travis.yml

```diff
@@ -1,11 +1,15 @@
-sudo: false
+sudo: required
 language: python
 python:
   - "2.7"
 
 services:
     - rabbitmq
 
+before_install:
+    - wget -O toxiproxy-2.0.0.deb https://github.com/Shopify/toxiproxy/releases/download/v2.0.0/toxiproxy_2.0.0_amd64.deb
+    - sudo dpkg -i toxiproxy-2.0.0.deb
```

Comment:
i didn't mean _using_ the library (i saw you commented about it not being ready). i meant using their approach to travis setup